### PR TITLE
unlisted authors

### PR DESCRIPTION
**Pull request by derrickdso**
The deprecated phila_author field needs to be hidden rather than removed. When its deleted, it returns a single string of the first item in the array of items that should be returned. 

ex returned when hidden
array(2) { [0]=> string(3) "180" [1]=> string(3) "304" }

ex returned when deleted
"180"

### CODE REVIEW

**Comment by derrickdso** [comment]
https://www.wrike.com/workspace.htm?acc=2747306#folder/884002094/board?overlayEntityId=849386919&sortOrder=10&spaceId=453350584&viewId=81744461

**Review by m-atia — APPROVED** [review]
works great on my local

File: wp/wp-content/plugins/phila.gov-customization/admin/meta-boxes/class-phila-gov-post.php [hunk 1]
```diff
@@ -28,10 +28,34 @@ function register_meta_boxes_posts($meta_boxes){
...
+          'sort_clone' => true,
+          'add_button' => '+ Add author',
+          'fields' => array(
+            array(
+              'name' => 'Additional author',
+              'id'   => 'phila_additional_author',
+              'type' => 'text',
+              'desc'  => 'The author of this post will be listed first, with authors chosen here appearing after, in the order listed.',
+              'size' => 50,
+              'columns' => 12,
```
Comment (karissademi) [inline wp/wp-content/plugins/phila.gov-customization/admin/meta-boxes/class-phila-gov-post.php:55]: Let's remove the 'columns' => 12 here. It helps clean up the admin by keeping the the checkbox on the same line as the label. 
Screen Shot 2022-09-08 at 5 44 00 PM

**Review by karissademi — APPROVED** [review]
Super minor thing. This is great!

**Pull request merged**